org.flightgear.FlightGear.yaml

```
@@ -108,13 +121,13 @@ modules:
     sha256: 277e817a39edb917976441a808b750ee64be38e077897d0d77c55dfc8eba9640
     dest: fgdata
   build-commands:
-  - mkdir --parents /app/share/flightgear
-  - mv fgdata/* /app/share/flightgear/
-    
+  - mkdir --parents ~/.fgfs
+  - mv fgdata/* ~/.fgfs/
```

The build fails with the same error. Try this.

```suggestion
  - mv fgdata ~/.fgfs
```